Package suggested git script with Python package (#15)
acsone/git-autoshare: README.rst

    $ pipx upgrade git-autoshare

This will also install a simple script to transparently replace your calls to ``git clone`` with calls to

> ``git autoshare-clone``. To enable it, set the `GIT_AUTOSHARE` variable in your shell to a positive value.

Reviewer: ```suggestion
``git autoshare-clone``. To enable it, export `GIT_AUTOSHARE=1` variable in your shell.
```
